x-pack/plugins/cross_cluster_replication/public/app/_app.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * 1. Encapsulate this fix to avoid modifying global styling.
+ */
+.ccrPageContent {
+  max-width: 1000px !important;
+  width: 100% !important;
+
+  .euiText .euiButton--warning {
+    color: $euiColorWarning; // [1]
+  }
+}
+
+.ccrFollowerIndicesFormRow {
```

Comment:
Thank you for following the prefix formatting. 

I know it may seem overkill for the few items you have in here, but we are trying to establish a pattern of creating SASS files next to the component they're altering and using more `_index.scss` files in each folder to import individual files.

The tendency before has been to create one file because there are very few custom styles, however, these can and have quickly ballooned to become one massive file that is hard to parse. If we can start with the above pattern early on, I think we can maintain it much better.

Reply:
I should also say that I'm really happy to see that in a pretty size-able feature PR with over 200 files and 10k lines of code it only has 3 custom styles 😄. **That** is pretty awesome!